srtcore/core.cpp

```diff
@@ -238,6 +238,14 @@ CUDT::CUDT(CUDTSocket* parent): m_parent(parent)
 
     m_pCache = NULL;
 
+    // This is in order to set it ANY kind of initial value, however
+    // this value should not be used when not connected and should be
+    // updated in the handshake. When this value is 0, it means that
+    // packets shall not be sent, as the other party doesn't have a
+    // room to receive and store it. Therefore this value should be
+    // overridden before any sending happens.
+    //m_iFlowWindowSize = 0;
+
```

Comment:
Commented code `m_iFlowWindowSize = 0`. Intended?

Reply:
No. Mistaken part of the test when I got a segfault :)